src/CodeGen_X86.cpp

```diff
@@ -392,6 +393,7 @@ void CodeGen_X86::visit(const Call *op) {
 
         // Left shift of negative integers is broken in some cases in
         // avx2: https://llvm.org/bugs/show_bug.cgi?id=27730
+        // Bug was fixed in LLVM r271796
 
         // It needs to be normalized to a 32-bit shift, because avx2
         // doesn't have a narrower version than that. We'll just do
```

Comment:
The comment below this line suggests that this produced better assembly, is that still the case?